terminal.go

```diff
@@ -881,21 +885,24 @@ func (g *LightningTerminal) startInternalSubServers(
 	}
 
 	log.Infof("Starting LiT account service")
-	err = g.accountService.Start(
-		g.lndClient.Client, g.lndClient.Router,
-		g.lndClient.ChainParams,
-	)
-	if err != nil {
-		log.Errorf("error starting account service: %v, disabling "+
-			"account service", err)
-		g.statusMgr.SetErrored(subservers.ACCOUNTS, err.Error())
-	} else {
-		g.statusMgr.SetRunning(subservers.ACCOUNTS)
+	if g.cfg.AccountsMode != ModeDisable {
+		err = g.accountService.Start(
+			g.lndClient.Client, g.lndClient.Router,
+			g.lndClient.ChainParams,
+		)
+		if err != nil {
+			log.Errorf("error starting account service: %v, "+
+				"disabling account service", err)
+			g.statusMgr.SetErrored(subservers.ACCOUNTS, err.Error())
+		} else {
+			g.statusMgr.SetRunning(subservers.ACCOUNTS)
+		}
 	}
-	// Even if we error on accountService.Start, we still want to mark the
-	// service as started so that we can properly shut it down in the
-	// shutdownSubServers call.
-	g.accountServiceStarted = true
+	// Even if we error on accountService.Start, or if the accounts service
+	// is disabled, we still want to mark that the service should be stopped
+	// so that we  can properly shut it down (canceling the ctx, and the
+	// accounts store) in the shutdownSubServers call.
+	g.accountServiceShouldBeStopped = true
```

Comment:
do we need it to be stopped if the accounts mode is disabled though? 

I also think the previous name for the variable is fine btw

Reply:
Hmm yeah agree that this is just strange. Perhaps we should just close everything in the `accountService` immediately if the service is disabled :)!

Reply:
Added that we now stop the account service directly if it is disabled instead!